### PR TITLE
Corrected layout for top row keys

### PR DESCRIPTION
**Pull request by newbold**
The original version reflected 14 columns and omitted KC_GRV, the 15th key on the top row. This revision adds support for the 15h key, and also updates the default (Tsangan) layout to include the HHKB layer keys.

## Description

1. Changed `MATRIX_COLS` from `14` to `15` in `config.h`
2. Updated layouts in `1up60hte.h` to reflect the missing key in the top row
3. Updated `keymaps/default` and `keymaps/hhkb` to reflect the missing key; updated the default keymap to include the HHKB layer keys

## Types of Changes

- [ ] Core
- [ ] Bugfix
- [ ] New feature
- [ ] Enhancement/optimization
- [x] Keyboard (addition or update)
- [x] Keymap/layout/userspace (addition or update)
- [ ] Documentation

## Issues Fixed or Closed by This PR

N/A

## Checklist

- [x] My code follows the code style of this project.
- [ ] My change requires a change to the documentation.
- [ ] I have updated the documentation accordingly.
- [x] I have read the [**CONTRIBUTING** document](https://docs.qmk.fm/#/contributing).
- [ ] I have added tests to cover my changes.
- [x] I have tested the changes and verified that they work and don't break anything (as well as I can manage).

### CODE REVIEW

File: keyboards/1upkeyboards/1up60hte/config.h [hunk 1]
```diff
@@ -30,7 +30,7 @@ along with this program.  If not, see <http://www.gnu.org/licenses/>.
 
 /* key matrix size */
 #define MATRIX_ROWS 5
-#define MATRIX_COLS 14
+#define MATRIX_COLS 15
```
Comment (drashna) [inline keyboards/1upkeyboards/1up60hte/config.h]: The column pins define below is missing the 15th pin.

**Comment by newbold** [comment]
@noroadsleft You're right; I reverted config.h and moved the key in 1up60hte.h. Flashed the board with these changes and all is well. Thanks!

File: keyboards/1upkeyboards/1up60hte/1up60hte.h [hunk 2]
```diff
@@ -20,11 +20,11 @@ along with this program.  If not, see <http://www.gnu.org/licenses/>.
 #include "quantum.h"
 
 #define LAYOUT_all( \
```
Comment (mechmerlin) [inline keyboards/1upkeyboards/1up60hte/1up60hte.h]:
```suggestion
#define LAYOUT_tsangan( \
```

File: keyboards/1upkeyboards/1up60hte/info.json [hunk 3]
```diff
@@ -0,0 +1,15 @@
+{
+    "keyboard_name": "1up60hte", 
+    "url": "https://www.1upkeyboards.com/shop/controllers/1up-rgb-60-pcb-hte/", 
+    "maintainer": "1upkeyboards", 
+    "width": 15, 
+    "height": 5, 
+    "layouts": {
+        "LAYOUT_all": {
```
Comment (mechmerlin) [inline keyboards/1upkeyboards/1up60hte/info.json]:
```suggestion
        "LAYOUT_tsangan": {
```

File: keyboards/1upkeyboards/1up60hte/keymaps/default/keymap.c [hunk 4]
```diff
@@ -17,21 +17,19 @@ along with this program.  If not, see <http://www.gnu.org/licenses/>.
 #include QMK_KEYBOARD_H
 
 const uint16_t PROGMEM keymaps[][MATRIX_ROWS][MATRIX_COLS] = {
-
 	[0] = LAYOUT_all(
```
Comment (mechmerlin) [inline keyboards/1upkeyboards/1up60hte/keymaps/default/keymap.c]:
```suggestion
	[0] = LAYOUT_tsangan(
```

File: keyboards/1upkeyboards/1up60hte/keymaps/default/keymap.c [hunk 5]
```diff
@@ -17,21 +17,19 @@ along with this program.  If not, see <http://www.gnu.org/licenses/>.
...
-		KC_LSFT, KC_Z, KC_X, KC_C, KC_V, KC_B, KC_N, KC_M, KC_COMM, KC_DOT, KC_SLSH, KC_LSFT, KC_NO,
-		KC_LCTL, KC_LGUI, KC_LALT, KC_SPC, KC_LALT, KC_LGUI, KC_LCTL, KC_BSPC),
-
+		KC_ESC,  KC_1,    KC_2,    KC_3,    KC_4,    KC_5,    KC_6,    KC_7,    KC_8,     KC_9,    KC_0,    KC_MINS, KC_EQL,  KC_BSLS, KC_GRV,
+		KC_TAB,  KC_Q,    KC_W,    KC_E,    KC_R,    KC_T,    KC_Y,    KC_U,    KC_I,     KC_O,    KC_P,    KC_LBRC, KC_RBRC, KC_BSPC,
+		KC_LCTL, KC_A,    KC_S,    KC_D,    KC_F,    KC_G,    KC_H,    KC_J,    KC_K,     KC_L,    KC_SCLN, KC_QUOT, KC_ENT,
+		KC_LSFT, KC_Z,    KC_X,    KC_C,    KC_V,    KC_B,    KC_N,    KC_M,    KC_COMM,  KC_DOT,  KC_SLSH, KC_LSFT, MO(1),
+		KC_LCTL, KC_LALT, KC_LGUI, KC_SPC,  KC_LGUI, KC_LALT, KC_LCTL),
+		
 	[1] = LAYOUT_all(
```
Comment (mechmerlin) [inline keyboards/1upkeyboards/1up60hte/keymaps/default/keymap.c]:
```suggestion
	[1] = LAYOUT_tsangan(
```

File: keyboards/1upkeyboards/1up60hte/keymaps/hhkb/keymap.c [hunk 6]
```diff
@@ -17,20 +17,19 @@ along with this program.  If not, see <http://www.gnu.org/licenses/>.
 #include QMK_KEYBOARD_H
 
 const uint16_t PROGMEM keymaps[][MATRIX_ROWS][MATRIX_COLS] = {
-
 	[0] = LAYOUT_all(
```
Comment (mechmerlin) [inline keyboards/1upkeyboards/1up60hte/keymaps/hhkb/keymap.c]:
```suggestion
	[0] = LAYOUT_tsangan(
```

File: keyboards/1upkeyboards/1up60hte/keymaps/hhkb/keymap.c [hunk 7]
```diff
@@ -17,20 +17,19 @@ along with this program.  If not, see <http://www.gnu.org/licenses/>.
...
-		KC_LSFT, KC_Z, KC_X, KC_C, KC_V, KC_B, KC_N, KC_M, KC_COMM, KC_DOT, KC_SLSH, KC_LSFT, MO(1),
-		KC_LCTL, KC_LGUI, KC_LALT, KC_SPC, KC_LALT, KC_LGUI, KC_LCTL, KC_BSPC),
-
+		KC_ESC,  KC_1,    KC_2,    KC_3,    KC_4,    KC_5,    KC_6,    KC_7,    KC_8,     KC_9,    KC_0,    KC_MINS, KC_EQL,  KC_BSLS, KC_GRV,
+		KC_TAB,  KC_Q,    KC_W,    KC_E,    KC_R,    KC_T,    KC_Y,    KC_U,    KC_I,     KC_O,    KC_P,    KC_LBRC, KC_RBRC, KC_BSPC,
+		KC_LCTL, KC_A,    KC_S,    KC_D,    KC_F,    KC_G,    KC_H,    KC_J,    KC_K,     KC_L,    KC_SCLN, KC_QUOT, KC_ENT,
+		KC_LSFT, KC_Z,    KC_X,    KC_C,    KC_V,    KC_B,    KC_N,    KC_M,    KC_COMM,  KC_DOT,  KC_SLSH, KC_LSFT, MO(1),
+		KC_NO,   KC_LALT, KC_LGUI, KC_SPC,  KC_LGUI, KC_LALT, KC_NO),
+		
 	[1] = LAYOUT_all(
```
Comment (mechmerlin) [inline keyboards/1upkeyboards/1up60hte/keymaps/hhkb/keymap.c]:
```suggestion
	[1] = LAYOUT_tsangan(
```

**Comment by newbold** [comment]
@mechmerlin Agreed, and done!

**Comment by drashna** [comment]
Thanks!

**Pull request merged**